### PR TITLE
fix tidb-debug docker build

### PR DESCRIPTION
**Pull request by cofyc**
### What problem does this PR solve? 

```
# wget http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz
...
# tar -xvf tidb-latest-linux-amd64.tar.gz 
tidb-v3.0.12-linux-amd64/
tidb-v3.0.12-linux-amd64/bin/
tidb-v3.0.12-linux-amd64/bin/pd-server
tidb-v3.0.12-linux-amd64/bin/tidb-ctl
tidb-v3.0.12-linux-amd64/bin/reparo
tidb-v3.0.12-linux-amd64/bin/tikv-ctl
tidb-v3.0.12-linux-amd64/bin/drainer
tidb-v3.0.12-linux-amd64/bin/binlogctl
tidb-v3.0.12-linux-amd64/bin/arbiter
tidb-v3.0.12-linux-amd64/bin/etcdctl
tidb-v3.0.12-linux-amd64/bin/pd-ctl
tidb-v3.0.12-linux-amd64/bin/tikv-server
tidb-v3.0.12-linux-amd64/bin/tidb-server
tidb-v3.0.12-linux-amd64/bin/pd-recover
tidb-v3.0.12-linux-amd64/bin/pump
```

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: misc/images/tidb-debug/Dockerfile [hunk 1]
```diff
@@ -27,8 +27,8 @@ RUN yum update -y && yum install -y \
 
 RUN wget -q http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz \
     && tar xzf tidb-latest-linux-amd64.tar.gz \
-    && mv tidb-latest-linux-amd64/bin/* /usr/local/bin/ \
```
Comment (DanielZhangQD) [inline misc/images/tidb-debug/Dockerfile:30]: What's this dockerfile used for?
The definition of `latest` has changed to the latest stable release and `nightly` now is the tag for the original latest daily build.
Do we need to change to nightly here?

Comment (cofyc) [reply misc/images/tidb-debug/Dockerfile:30]: cc @aylei

Comment (cofyc) [reply misc/images/tidb-debug/Dockerfile:30]: this is used for debug running tidb clusters in Kubernetes, it's safe to use the stable release.

Comment (aylei) [reply misc/images/tidb-debug/Dockerfile:30]: Yes

**Pull request merged**